src/libraries/System.Collections.Immutable/src/System/Collections/Immutable/MultiSet.cs

```diff
@@ -0,0 +1,84 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Collections.Generic;
+using System.Runtime.CompilerServices;
+using System.Runtime.InteropServices;
+
+namespace System.Collections.Immutable
+{
+    internal class MultiSet<T>
```

Comment:
I think the MultiSet class is trivial enough that it can be moved back into the original source file. Also, it can probably be made a struct now.

Reply:
Fixed.